lib/karto_sdk/include/karto_sdk/Mapper.h

```diff
@@ -1996,17 +1996,17 @@ class KARTO_EXPORT Mapper : public Module
    *
    * @return true if the scan was added successfully, false otherwise
    */
```

Comment:
For all: linting is needed. Spaces between `*` and `=`s

Reply:
See styling of surrounding functions

Reply:
Addressed linting issues.